qiskit/circuit/random/utils.py

```diff
@@ -21,7 +21,7 @@
 
 
 def random_circuit(
-    num_qubits, depth, max_operands=4, measure=False, conditional=False, reset=False, seed=None
+    num_qubits, depth, num_operand_distribution: dict = None,  max_operands=None, measure=False, conditional=False, reset=False, seed=None
```

Comment:
In general, it is better not to change the order of the positional arguments. Otherwise, you might accidentally break callers like `random_circuit(2, 4, 10)`